### PR TITLE
feat: Node 14 and file upload

### PR DESCRIPTION
**Pull request by vinnyjth**
Testing instructions

```
nvm use 14
BRANCH=node-14-and-file-upload yarn api
BRANCH=node-14-and-file-upload yarn app
```

Then upload a profile picture :)

### CODE REVIEW

**Comment by vinnyjth** [comment]
Companion PR: https://github.com/ApollosProject/apollos-templates/pull/443

File: .nvmrc [hunk 1]
```diff
@@ -1 +1 @@
-10.19.0
+14.x.x
```
[the remark this answers is not in the document]
Comment (vinnyjth) [reply .nvmrc]: done

File: packages/apollos-server-core/package.json [hunk 2]
```diff
@@ -35,10 +35,12 @@
     "bull-board": "^0.1.1",
     "express-basic-auth": "^1.2.0",
     "graphql-parse-resolve-info": "^4.4.5",
+    "graphql-upload": "^12.0.0",
     "lodash": "^4.17.11"
   },
   "peerDependencies": {
-    "apollo-server": "^2.11.0"
+    "apollo-server": "^2.11.0",
```
Comment (vinnyjth) [inline packages/apollos-server-core/package.json:42]:
```suggestion
    "apollo-server": "^2.11.0"
```

File: packages/apollos-server-core/src/index.js [hunk 3]
```diff
@@ -206,6 +206,8 @@ export const createMiddleware = (data) => ({ app, context, dataSources }) => {
 
   const getContext = createContextGetter({ context, dataSources });
 
+  // Upload middleware.
```
Comment (vinnyjth) [inline packages/apollos-server-core/src/index.js:209]:
```suggestion
```

File: packages/apollos-server-core/src/index.js [hunk 4]
```diff
@@ -206,6 +206,7 @@ export const createMiddleware = (data) => ({ app, context, dataSources }) => {
 
   const getContext = createContextGetter({ context, dataSources });
 
+
```
Comment (vinnyjth) [inline packages/apollos-server-core/src/index.js:209]:
```suggestion
```

**Comment by redreceipt** [comment]
error running `yarn api`

Screen Shot 2021-08-04 at 11 13 18 AM

**Comment by redreceipt** [comment]
sike, disregard...

**Comment by redreceipt** [comment]
the bundler isn't starting when trying to pull up the dev sim

Screen Shot 2021-08-04 at 11 41 44 AM

**Comment by vinnyjth** [comment]
@redreceipt It's likely that your bundler is using `node12` (since it opened in a different context)

**Comment by vinnyjth** [comment]
That error occurs when the version of `node` that installed `node_module/sharp` is different than the version of `node` running `node_modules/sharp`

**Comment by redreceipt** [comment]
> That error occurs when the version of `node` that installed `node_module/sharp` is different than the version of `node` running `node_modules/sharp`

oh shoot i know why, my default version is 16 and the app branch is locked to 14. just a sec...

**Review by redreceipt — APPROVED** [review]

**Pull request merged**